provider/manual/environ.go

```diff
@@ -258,7 +258,7 @@ if [ $stopped -ne 1 ]; then
     service %s stop
 fi
 rm -f /etc/init/juju*
-rm -f /etc/systemd/system/juju*
+rm -f /etc/systemd/system{,/multi-user.target.wants}/juju*
```

Comment:
Does this have anything to do with IPv6 or is it just a drive by that you encountered. Should we be tagging a bug about this so people can know where/when it was fixed?